### PR TITLE
DM-20554 Log or store if ip_diffim running in deconvolution case

### PR DESCRIPTION
**Pull request by kgabor**

### CODE REVIEW

File: python/lsst/ip/diffim/makeKernelBasisList.py [hunk 1]
```diff
@@ -232,6 +252,8 @@ def generateAlardLuptonBasisList(config, targetFwhmPix=None, referenceFwhmPix=No
         # http://iopscience.iop.org/0266-5611/26/8/085002  Equation 40
 
         # Use specializations for deconvolution
+        logger.debug("Target psf fwhm is the greater, deconvolution mode")
```
Comment (isullivan) [inline python/lsst/ip/diffim/makeKernelBasisList.py]: I think this should be a higher level of logging output than `debug`. Maybe even `warn`, but at least `info`.

File: python/lsst/ip/diffim/psfMatch.py [hunk 2]
```diff
@@ -376,25 +376,26 @@ def setDefaults(self):
...
-        doc="Default sigma values in pixels of Gaussian kernel base functions. "
+        doc="Default sigma values in pixels of base Gaussians. "
             "List length must be `alardNGauss`.",
         default=(0.7, 1.5, 3.0),
     )
     alardGaussBeta = pexConfig.Field(
         dtype=float,
-        doc="""Multiplier of Gaussian sigmas between adjacent elements of the kernel basis list.""",
+        doc="Scaling multiplier of base Gaussian sigmas for automated sigma determination "
+            "(scaleByFwhm = True case).",
```
Comment (isullivan) [inline python/lsst/ip/diffim/psfMatch.py]: scaleByFwhm should be in back-ticks, like `alardNGauss` in the entry above. I would also write this as "Used if `scaleByFwhm`=True". Or, something like that if that's inaccurate.

Comment (kgabor) [reply python/lsst/ip/diffim/psfMatch.py]: Reworded.

File: python/lsst/ip/diffim/makeKernelBasisList.py (hunk 1 above)
Comment (kgabor) [reply python/lsst/ip/diffim/makeKernelBasisList.py]: Increased to `info`.

**Pull request merged**